src/Oro/Bundle/EntityBundle/Controller/EntitiesController.php

```diff
@@ -309,6 +321,16 @@ public function updateAction(Request $request, $entityName, $id)
     }
 
     /**
+     * @param string $entityClass
+     *
+     * @return bool
+     */
+    protected function isEnumClass($entityClass)
+    {
+        return strpos($entityClass, ExtendHelper::ENTITY_NAMESPACE . 'EV_') !== false;
```

Comment:
Please change to verification by enum code - for enums it should be present in config, e.g. run console command
./app/console oro:entity-config:debug "Extend\Entity\EV_Lead_Source